src/clj/rems/api/resources.clj

```diff
@@ -38,19 +39,21 @@
 
     (GET "/" []
       :summary "Get resources"
-      :roles #{:owner :handler}
+      :roles #{:owner :organization-owner :handler}
       :query-params [{disabled :- (describe s/Bool "whether to include disabled resources") false}
                      {archived :- (describe s/Bool "whether to include archived resources") false}]
       :return Resources
       (ok (resource/get-resources (merge (when-not disabled {:enabled true})
-                                         (when-not archived {:archived false})))))
+                                         (when-not archived {:archived false}))
+                                  (getx-user-id))))
 
     (GET "/:resource-id" []
       :summary "Get resource by id"
-      :roles #{:owner :handler}
+      :roles #{:owner :organization-owner :handler}
       :path-params [resource-id :- (describe s/Int "resource id")]
       :return Resource
-      (ok (resource/get-resource resource-id)))
+      (ok (or (resource/get-resource resource-id (getx-user-id))
+              (throw-forbidden))))
```

Comment:
Why not throw in `get-resource` already?

Reply:
I thought it's more API level stuff than db level stuff to decide what to show exactly when the resource is not available.

Reply:
I think it's also how it's done elsewhere in the API, e.g., /api/applications/[id] endpoint:

```
    (GET "/:application-id" []
      :summary "Get application by `application-id`"
      :roles #{:logged-in}
      :path-params [application-id :- (describe s/Int "application id")]
      :responses {200 {:schema Application}
                  404 {:schema s/Str :description "Not found"}}
      (if-let [app (applications/get-application (getx-user-id) application-id)]
        (ok app)
        (api-util/not-found-json-response)))

Reply:
perhaps the weird part is `(ok (or (get-it) (throw-forbidden)))`, so how about:
```
(if-let [thing (get-it)]
  (ok thing)
  (forbidden-response)))
```

Reply:
Done.